.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.2.14
+current_version = 1.2.15
```

Comment:
Please do not touch version numbers, I update it automatically wen I'm doing releases.
```suggestion
current_version = 1.2.14
```